targets/lenovo-x1-carbon.nix

```diff
@@ -68,17 +68,17 @@
       ({pkgs, ...}: {
         ghaf.graphics.weston.launchers = [
           {
-            path = "${pkgs.waypipe}/bin/waypipe ssh -i ${pkgs.waypipe-ssh}/keys/waypipe-ssh -o StrictHostKeyChecking=no 192.168.101.5 chromium --enable-features=UseOzonePlatform --ozone-platform=wayland";
+            path = "${pkgs.waypipe}/bin/waypipe ssh -i ${pkgs.waypipe-ssh}/keys/waypipe-ssh -o StrictHostKeyChecking=no chromium-vm.ghaf chromium --enable-features=UseOzonePlatform --ozone-platform=wayland";
```

Comment:
The same goes here on proposal to rename this to `browser-vm` - I'm also fine on scoping this out of this PR and creating another issue for renaming. This refactoring creates a generic mechanism for appvms naming.

Reply:
Let's postpone naming for now.